lib/CMakeLists.txt

```diff
@@ -31,19 +34,19 @@ if(WITH_XKB)
 endif()
 
 add_subdirectory(xpp)
-list(APPEND libs ${XPP_LIBRARIES})
-list(APPEND dirs ${XPP_INCLUDE_DIRS})
+add_library(xpp-interface INTERFACE)
```

Comment:
`xpp` already calls `add_library` itself [here](https://github.com/jaagr/xpp/blob/master/CMakeLists.txt#L173).
We could only do `list(APPEND libs xpp)`

Reply:
Good point. I think I was tripped up by the fact that they link privately against `${XPP_LIBRARIES}`, but on a second look, I think that should be fine.